html/feed.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
The file structure here does not meet the expectations. You should have this `/feed/index` for feed page and `/profile/index.html` for profile page